.eslintrc

```diff
@@ -16,7 +16,7 @@
     "new-cap": 0,
     "consistent-return": 0,
     "no-param-reassign": 0,
-    "no-confusing-arrow": 0,
```

Comment:
I believe you can include the new guide without removing `no-confusing-arrow`. This is still a valid check.